js/src/Figure.js

```diff
@@ -552,15 +564,21 @@ var Figure = widgets.DOMWidgetView.extend({
                       .attr({"y": (legend_height + 2) / 2.0,
                              "x": (-0.5 * (legend_height + 2))})
                       .attr("width", (max_label_len + 2) + "em")
-                      .attr("height", (count * (legend_height + 2)))
-                      .style({"fill": "none"});
+                      .attr("height", (count * (legend_height + 2)));
+                      
                 }
                 max_label_len = (legend_location === "top-right" ||
                                  legend_location === "right" ||
                                  legend_location === "bottom-right") ? -(max_label_len + 2) : 1;
                 var em = 16;
                 legend_g.attr("transform", "translate(" + String(coords[0] + max_label_len * em) + " " +
                                                           String(coords[1]) + ") ");
+
+                var legend_rect = legend_g.selectAll("rect");
```

Comment:
@madridnick Since you don't reuse this variable, you can chain the two statements without an assignment